pkgs/by-name/pl/pluginval/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  gtk3,
+  freetype,
+  fontconfig,
+  xorg,
+  curl,
+  webkitgtk_4_1,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libdatrie,
+  libXdmcp,
+  libdeflate,
+  lerc,
+  xz,
+  libwebp,
+  libxkbcommon,
+  libepoxy,
+  libXtst,
+  sqlite,
+  ladspaH,
+}:
+stdenv.mkDerivation rec {
+  pname = "pluginval";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "Tracktion";
+    repo = "pluginval";
+    rev = "v${version}";
+    sha256 = "sha256-j4Lb3pcw0931o63OvTTaIm2UzvYDIjmnaCXGvKB4gwM=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+  buildInputs = [
+    alsa-lib
+    gtk3
+    freetype
+    fontconfig
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXinerama
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    curl
+    webkitgtk_4_1
+    libsysprof-capture
+    pcre2
+    util-linux
+    libselinux
+    libsepol
+    libthai
+    libdatrie
+    libXdmcp
+    libdeflate
+    lerc
+    xz
+    libwebp
+    libxkbcommon
+    libepoxy
+    libXtst
+    sqlite
+    ladspaH
+  ];
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Debug"
+  ];
+  installPhase = ''
+    install -Dm755 pluginval_artefacts/Debug/pluginval $out/bin/pluginval
+  '';
```

Comment:
Add `runHook preInstall` and `runHook postInstall` if you want to override the entire phase.

Also does this work for darwin platforms?  It looks like they have macOS application bundles (pluginval.app) that needs to be installed.

Reply:
I don't have a mac, so maybe I should change the platforms to linux, and if a mac user wants to use this package, they can adapt the pkg and do a PR?

Reply:
Ah, you adressed this later on, never mind.